builders/integrations/.pages

```diff
@@ -4,3 +4,4 @@ nav:
  - bridges
  - indexers
  - oracles
+ - wallets
```

Comment:
couldn't add a comment on the actual line, but is the translation for integrations the same as integrate?

Reply:
Yes, they are the same word in Chinese.